internal/cli/atlas/backup/snapshots/describe.go

```diff
@@ -0,0 +1,74 @@
+// Copyright 2020 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package snapshots
+
+import (
+	"github.com/mongodb/mongocli/internal/cli"
+	"github.com/mongodb/mongocli/internal/config"
+	"github.com/mongodb/mongocli/internal/flag"
+	"github.com/mongodb/mongocli/internal/output"
+	"github.com/mongodb/mongocli/internal/store"
+	"github.com/mongodb/mongocli/internal/usage"
+	"github.com/spf13/cobra"
+)
+
+const describeTemplate = `ID	SNAPSHOT TYPE	TYPE	DESC	EXPIRES AT
```

Comment:
I think we can leave `SNAPSHOT TYPE` to just `TYPE` also so far we prefer to use `DESCRIPTION` over `DESC`

Reply:
We have `snapshotType": "onDemand"` and `"type": "replicaSet"`. Which one should we show?

Reply:
I am sure if you are suggesting to remane `SNAPSHOT TYPE` to `TYPE` and remove the other `TYPE` 😕

Reply:
Oops sorry I did not see the `type": "replicaSet"` this is weird I would make `SNAPSHOT TYPE` -> `Type` and the `Type` -> `Cluster Type` since we are talking snapshots here but let's not go there this may end up being more confusing